smart-contracts/contracts/basic-vault/src/multitest/vault.rs

```diff
@@ -17,7 +17,7 @@ fn try_bond() {
     assert_eq!(
         err,
         VaultContractError::Std(cosmwasm_std::StdError::GenericErr {
-            msg: "alloc::string::String not found".to_string()
+            msg: "type: alloc::string::String; key: [69, 63, 71, 5F, 63, 68, 61, 6E, 6E, 65, 6C] not found".to_string()
```

Comment:
@arhamchordia What are these numbers?

Reply:
Keys are in bytes and it was trying to find this key in a map.

Reply:
These bytes translate to icq_channel